### PR TITLE
Add missing translations in the users panel

### PR DESCRIPTION
**Pull request by bieniu**
## Breaking change

None

## Proposed change

This PR adds missing translations in the users panel.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/users/dialog-add-user.ts [hunk 1]
```diff
@@ -85,7 +85,9 @@ export class DialogAddUser extends LitElement {
             required
             auto-validate
             autocapitalize="on"
-            error-message="Required"
+            error-message=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/config/users/dialog-add-user.ts:88]:
```suggestion
            .errorMessage=${this.hass.localize(
```

File: src/panels/config/users/dialog-add-user.ts [hunk 2]
```diff
@@ -99,7 +101,9 @@ export class DialogAddUser extends LitElement {
             auto-validate
             autocapitalize="none"
             @value-changed=${this._usernameChanged}
-            error-message="Required"
+            error-message=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/config/users/dialog-add-user.ts:104]:
```suggestion
            .errorMessage=${this.hass.localize(
```

File: src/panels/config/users/dialog-add-user.ts [hunk 3]
```diff
@@ -110,18 +114,19 @@ export class DialogAddUser extends LitElement {
             required
             auto-validate
             @value-changed=${this._passwordChanged}
-            error-message="Required"
+            error-message=${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/config/users/dialog-add-user.ts:117]:
```suggestion
            .errorMessage=${this.hass.localize(
```

File: src/translations/en.json [hunk 4]
```diff
@@ -1439,7 +1440,8 @@
             "name": "Name",
             "username": "Username",
             "password": "Password",
-            "create": "Create"
+            "create": "Create",
+            "error_required": "Required"
```
Comment (bramkragten) [inline src/translations/en.json:1444]: This could also be moved to generic

Comment (bieniu) [reply src/translations/en.json]: Done

**Review by bramkragten — APPROVED** [review]

**Pull request merged**